ros_ign_gazebo_demos/launch/gpu_lidar.launch

```diff
@@ -1,8 +1,8 @@
 <?xml version="1.0"?>
 <launch>
 
-  <include file="$(find ros_ign_gazebo_demos)/launch/ign_gazebo.launch">
-    <arg name="args" value="-r -v 3 $(find ros_ign_point_cloud)/examples/gpu_lidar.sdf"/>
+  <include file="$(find ros_ign_gazebo)/launch/ign_gazebo.launch">
+    <arg name="ign_args" value="-r -v 3 $(find ros_ign_point_cloud)/examples/gpu_lidar.sdf"/>
```

Comment:
`ros_ign_point_cloud` currently has a `COLCON_IGNORE`, so this demo doesn't work.